pkgs/development/python-modules/flit/default.nix

```diff
@@ -25,6 +25,7 @@ buildPythonPackage rec {
     repo = "flit";
     rev = version;
     hash = "sha256-iXf9K/xI4u+dDV0Zf6S08nbws4NqycrTEW0B8/qCjQc=";
+    name = "${pname}-${version}-source";
```

Comment:
I just tried it locally and that does not change the hash which is really surprising to me because I always thought it did. Only the store path changed.